src/Main.java

```diff
@@ -0,0 +1,363 @@
+import java.time.LocalDate;
+import java.util.*;
+
+public class Main {
+    public static void main(String[] args) {
+        //Задача2
+        //System.out.println(returnDayInMonth(Months.APRIL));
+        //Задача3
+        //System.out.println(differenceBetweenMinAndMaxElementsInArray(makeArrayWithRandomElements(6)));
+        //Задача4
+        //System.out.println(comparisonElementsTwoArrays(makeArrayWithRandomElements(2), makeArrayWithRandomElements(3)));
+        //Задача5
+        //System.out.println(searchMissedNumberFromZeroToNineInArray(new int[]{1, 0, 4, 9, 6, 5, 3, 8, 7, 2}));
+        //Задача6
+        //System.out.println(findMinElement(generateTwoDimensionalArray(5, 6)));
+        //Задача7
+        //printTwoDimensionalArray(swapMinAndMaxElementsInTwoDimensionalArray(generateTwoDimensionalArray(4, 4)));
+        //swapMinAndMaxElementsInTwoDimensionalArray2(generateTwoDimensionalArray(4, 4));
+        //Задача8
+        //printTwoDimensionalArray(replaceOddOnEvenElementInTwoDimensionalArray(generateTwoDimensionalArray(4, 4)));
+        //Задача9
+        /*try {
+            System.out.println(multiplyDiagonalElementsRelativeEnteredElement(generateTwoDimensionalArray(5, 5), 9));
+        }
+        catch (NoElementInArray e) {
+            System.out.println(e.getMessage());
+        }   */
+        //Задача10
+        //System.out.println(hasDatePassed2(Holidays2.YESTERDAY));
+
+    }
+
+    public static int returnDayInMonth (Months months){
+        int a = 0;
+        switch (months) {
+            case JANUARY:
+                a = 31;
+                break;
+            case FEBRUARY:
+                a = 28;
+                break;
+            case MARCH:
+                a = 31;
+                break;
+            case APRIL:
+                a = 30;
+                break;
+            case MAY:
+                a = 31;
+                break;
+            case JUNE:
+                a = 30;
+                break;
+            case JULY:
+                a = 31;
+                break;
+            case AUGUST:
+                a = 31;
+                break;
+            case SEPTEMBER:
+                a = 30;
+                break;
+            case OCTOBER:
+                a = 31;
+                break;
+            case NOVEMBER:
+                a = 30;
+                break;
+            case DECEMBER:
+                a = 31;
+                break;
+        }
+        return a;
+    }
+
+    public static int[] makeArrayWithRandomElements (int amountOfElemets) {
+        int[] array = new int[amountOfElemets];
+        for (int i = 0; i < amountOfElemets; i++) {
+            array[i] = (int) (Math.random() * 100);
```

Comment:
why math random? we agreed to use Random

Reply:
and random generator should be global variable

Reply:
reworked